compiler/src/main/scala/dx/translator/cwl/ProcessTranslator.scala

```diff
@@ -80,10 +80,30 @@ import spray.json._
 import java.nio.file.{Path, Paths}
 import scala.util.Try
 
-case class CwlSourceCode(source: Path) extends SourceCode {
+case class CwlSourceCode(source: Path, target: Vector[String] = Vector.empty) extends SourceCode {
```

Comment:
If you could add a help string with a short description on the class (what the inputs are)?

Reply:
Added. Thanks!

Reply:
Thank you!